src/js/adjustHeights.js

```diff
@@ -0,0 +1,39 @@
+import $ from "jquery";
+window.jQuery = $; window.$ = $;
+
+const setHeightsArray = [
+    // { className: 'flex-col--2__h1', parentId: 'naprawa'},
+    // { className: 'flex-col--2__p--1', parentId: 'naprawa'},
+    // { className: 'flex-col--2__p--2', parentId: 'naprawa'},
+    // { className: 'flex-col--2__list', parentId: 'naprawa'},
+];
+
+
+//adjust height of a div to the highest child of all considered divs
+export function setHeights() {
+    setHeightsArray.forEach(arrayEl => {
+        const parentElement = document.getElementById(arrayEl.parentId);
+        const elements = parentElement.querySelectorAll('.' + arrayEl.className);
+        let maxHeight = 0;
+        
+        elements.forEach(el => {
+            let childHeight = el.children[0].scrollHeight;
+            
+            if (maxHeight < childHeight) {
+                maxHeight = childHeight;
+            }
+        });
```

Comment:
Just wonder if we want to use [Lodash](https://lodash.com/docs/4.17.10) in our code.  If yes, we could use

```js
const heights = elements.map(it => it.children[0].scrollHeight);
const maxHeight = _.max(heights);
```